site/topics/control-logic/counting_output.hex

```diff
@@ -0,0 +1,17 @@
+v2.0 raw
```

Comment:
this one is
0 --- Load into A the value 0
1 --- Load into B the value 1
2 --- Add B to A and save in A
3 --- Write A to address 15
4 --- Output unsigned into from address 15
5 --- Jump to 0 (load into program counter the value 0)
6
7
8
9
10
11
12
13
14
15